app/proxy/proxy.go

```diff
@@ -79,7 +80,9 @@ func Handle(w http.ResponseWriter, r *http.Request) {
 	if cache.IsOnRequest(r) {
 		qCache = cache.FromRequest(r)
 	}
-	c := query.NewCallerWithCache(sdkAddress, userID, qCache)
+	c := query.NewCaller(sdkAddress, userID)
+	c.Cache = qCache
+	c.DB = boil.GetDB()
```

Comment:
they are not always required (e.g. in tests). i could put them in the initializer, but then you have to pass `nil` all the time. i'm open to it though. it would make the dependencies more explicit.